README.md

```diff
@@ -1,27 +1,47 @@
 Cloudinary Android SDK
 ======================
-
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
-
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software.
-Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements.
-Images are seamlessly delivered through a fast CDN, and much much more.
-
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
-
-Cloudinary provides URL and HTTP based APIs that can be easily integrated with any Web development framework.
-
-## Requirements
-The library requires Android version 4.0.3 (Ice Cream Sandwich) and up.
-
-## Gradle Integration
+[![Build Status](https://api.travis-ci.com/cloudinary/cloudinary_android.svg?branch=master)](https://app.travis-ci.com/github/cloudinary/cloudinary_android)
+
+## About
+The Cloudinary Android SDK allows you to quickly and easily integrate your application with Cloudinary.
+Effortlessly optimize and transform your cloud's assets.
+
+### Additional documentation
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Android SDK Guide](https://cloudinary.com/documentation/android_integration).
+
+## Table of Contents
+- [Key Features](#key-features)
+- [Compatibility](#Version Support)
+- [Installation](#installation)
+- [Usage](#usage)
+    - [Setup](#Setup)
+    - [Transform and Optimize Assets](#Transforming and Optimizing Assets)
```

Comment:
This is a broken link, please see how it looks like:
https://github.com/cloudinary/cloudinary_android/blob/update/readme/README.md